### DISCUSSION TITLE
go:build comment without // +build comment error

### DISCUSSION THREAD

**Discussion by joyahamad074** [projectdiscovery/nuclei#2210] [Q&A] [answered]
When I try to install nuclei in my ubuntu with `go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest ` this command it shows the error like this `../go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.7.3/internal/runner/update.go:29:2: //go:build comment without // +build comment ../go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.7.3/cmd/nuclei/main.go:14:2: //go:build comment without // +build comment`

**Comment by ehsandeep** [comment, marked as answer]
@joyahamad074 This is because you have an old Go version installed on your system, make sure to update to 1.18 + before installing nuclei.
